easybuild/easyconfigs/l/LAMMPS/LAMMPS-11Aug2017-CrayGNU-17.08-cuda-8.0-pat-6.5.1.eb

```diff
@@ -0,0 +1,48 @@
+# Contributed by Luca Marsella and Victor Holanda Rusu (CSCS)
+easyblock = 'MakeCp'
+
+name = 'LAMMPS'
+version = '11Aug17'
+release = 'stable_11Aug2017'
+cudaversion =  '8.0'
+patversion = '6.5.1'
+versionsuffix = '-cuda-%s-pat-%s' % (cudaversion, patversion)
+
+homepage = 'http://lammps.sandia.gov/'
+description = "LAMMPS (Large-scale Atomic/Molecular Massively Parallel Simulator) is a classical molecular dynamics simulation code designed to run efficiently on parallel computers."
```

Comment:
description = """LAMMPS (Large-scale Atomic/Molecular Massively Parallel
Simulator) is a classical molecular dynamics simulation code designed to run
efficiently on parallel computers."""